DESCRIPTION

```diff
@@ -19,7 +19,8 @@ Imports:
     digest,
     R6,
     visNetwork,
-    methods
+    methods,
+    snippr
```

Comment:
{snippr} failed to install in R3.6.1 in DaVinci. It's only on Github?Suggest further testing on DaVinci before releasing.

Reply:
I just installed it!